src/DtoAbstract.php

```diff
@@ -163,25 +103,23 @@ public static function fromModels(Collection $models): Collection
      * @return $this
      * @throws \Exception
      */
-    public static function fromArray(array $data): DtoAbstract
+    public function populateFromArray(array $data): DtoAbstract
```

Comment:
Why not make this method and `populateFromModel` protected, and move the validation to `from`. That will prevent us to have `$this->validation && $this->validate();` in every override in DTO's.

Reply:
Can check it a bit later, for sure. Will get the current story merged first.